### PR TITLE
fixed/navbar covering dashboard

### LINKED ISSUE #1186: bug: Courses dashboard gets partially covered
**Describe the bug**
On opening the side navBar  the courses Dashboard get covered.

**To Reproduce**
1. Click on the show Nav button in the left div at the top.
2. It covers the courses dashboard.

**Expected behavior**
The complete courses dashboard should be visible even if the nav is open.

Screenshots or GIFs
Screenshot 2024-09-11 at 12 50 27 PM

### PR DESCRIPTION
**Pull request by spc-28**
### PR Fixes:
#1186 
The dashboard is visible even if the nav is open.

Resolves #1186

### Checklist before requesting a review
- [ *] I have performed a self-review of my code
- [ *] I assure there is no similar/duplicate pull request regarding same issue
Screenshot 2024-09-11 at 12 43 03 PM

### CODE REVIEW

**Comment by keshav-exe** [comment]
Can you elaborate this a bit more?

**Comment by spc-28** [comment]
@keshav-exe 
I Didn't get what do you mean by elaborate, do you mean to explain the issue in more detailed form or to explain the PR fixes in more elaborative way? 
I m new to open source, please guide me with what I m required to do.

**Comment by keshav-exe** [comment]
Hey it's okay, I just didn't get what you mentioned in the issue can you rephrase it please?

**Comment by spc-28** [comment]
@keshav-exe 
On the Home page of the website, when we Click on the Menu Button which is present on the top left corner of the page, it opens up the Menu bar. This Menu bar is overlapping the contents like the "Greeting" and the Courses present, which is impacting the user experience on the site. Here is an image for reference,
g1

**Comment by keshav-exe** [comment]
Thanks! Will fix it.

**Comment by devsargam** [comment]
Don't think this is a problem anymore. Can't seem to replicate as well
